lib/test/fundAddress.js

```diff
@@ -24,14 +24,14 @@ async function fundAddress(dapiClient, faucetAddress, faucetPrivateKey, address,
     .fee(668)
     .sign(faucetPrivateKey);
 
-  let { blocks: currentBlockHeight } = await dapiClient.getStatus();
+  let { blocks: currentBlockHeight } = await dapiClient.core.getStatus();
 
-  const transactionId = await dapiClient.sendTransaction(transaction.toBuffer());
+  const transactionId = await dapiClient.core.broadcastTransaction(transaction.toBuffer());
 
   const desiredBlockHeight = currentBlockHeight + 2;
 
   do {
-    ({ blocks: currentBlockHeight } = await dapiClient.getStatus());
+    ({ blocks: currentBlockHeight } = await dapiClient.core.getStatus());
```

Comment:
We need a function like `waitForBlocks(N)`. It should use `generate` for `regtest` and wait for blocks in other cases.